src/objects/RecurringDonationStatusMapping__mdt.object

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>Recurring Donation Status to State Mapping</description>
+    <fields>
+        <fullName>State__c</fullName>
+        <description>The state that the Recurring Donation Status will be mapped to.</description>
```

Comment:
@sam-knox , can you please review new Recurring Donation Status Mapping custom metadata SObject field description and help text?

Reply:
@RadSFDeveloper - Sure thing - I have a couple of questions here. I'm less familiar with custom metadata types. Will there be one record of this object for each mapping a user creates? For State__c, would it be more accurate to say that the State is mapped to a Status versus will be mapped to a Status? In other words, don't these records only exist once a mapping has been saved?

Reply:
If there are no new Status values, then no RD Status Mapping custom metadata type (CMT) record will exist, the RD2 will work fine since Active/Lapsed/Closed Status are mapped by default. When a new Status value exists and an admin saves mapping, records will be saved. Here is an example:

<img width="1138" alt="Screen Shot 2020-05-14 at 12 00 24 PM" src="https://user-images.githubusercontent.com/28743286/81974896-128f9e80-95db-11ea-9ead-1cc77806ed38.png">